app/assets/stylesheets/changelog/changelog.scss

```
@@ -0,0 +1,18 @@
+// Place all the styles related to the changelog/changelog controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.changelogentry-title {
```

One convention that I recently started following helps with CSS namespacing. 

```suggestion
.changelog-entry__title {
```

---

Updated